NineChronicles.Headless/GraphTypes/StateQuery.cs

```diff
@@ -346,7 +349,136 @@ public StateQuery()
                     return null;
                 }
             );
+            Field<NonNullGraphType<DecimalGraphType>>(
+                name: "arenaPercentageCalculator",
```

Comment:
I guess `stateQuery` might not be the best place for this sort of query because it seems very domain specific(i.e., PVP system) logic instead of simple state lookups. (but on the other hand, it isn't strong due to a lack of alternatives 😅 ) cc @planetarium/nine-chronicles

Reply:
May I suggest a new category called "simulatorQuery"? I also got a Stage/PVE one that is almost done. So both queries could be set under this new category.